products/kirby.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Kirby
+category: server-app
+iconSlug: kirby
+permalink: /kirby
+alternate_urls: 
+-   /getkirby
```

Comment:
We use alternate URLs for variations in spelling and alternate names. This seems needless.

```suggestion
```

Reply:
`getkirby` is used as handle all across the projects' ecosystem as ([the project website](https://getkirby.com), [Instagram](https://www.instagram.com/getkirby/), [Twitter](https://twitter.com/getkirby), [Mastodon](https://mastodon.social/@getkirby). That is why I added this here.

Reply:
I'm still not sure, but seeing as it won't conflict anywhere, and it causes us no issues - going ahead with this.